packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -142,7 +152,43 @@ export async function deployConfirmed(
   }
 }
 
-async function ensureNonUuidManagedExtensionsIds(
+function matchWebhooks(remoteSource: RemoteSource, extension: ExtensionInstance) {
+  const remoteVersionConfig = remoteSource.activeVersion?.config ?? remoteSource.draftVersion?.config
```

Comment:
should we remove the `draftVersion`? since `dev `won't work with webhook subscriptions